test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -0,0 +1,101 @@
+package mongodbtests
```

Comment:
this package contains functions that return test functions which will be called by the framework for commonly tested things